ztp/policygenerator/policyGen/policyHelper.go

```diff
@@ -58,3 +58,19 @@ func CheckNameLength(namespace string, name string) error {
 	}
 	return nil
 }
+
+// Create a new ObjectTemplate for the given resource with values
+// populated from the reference ACM template (pulled from
+// policyBuilder)
+func BuildObjectTemplateFor(resource generatedCR) (utils.ObjectTemplates, error) {
```

Comment:
Agree on the name. For the parameter I prefer to keep the structure which binds the PGT sourceFile with the built object. As we handle other parameters (eg enforcement type) the data is all available within this struct instead of passing multiple parameters to functions.

Reply:
Name changed.